tests/commands/test_unset.py

```diff
@@ -44,23 +43,17 @@ def test_unset(repo: Repo, asset: str) -> None:
     assert ret.returncode == 0
 
     # verify changes, and the repository clean
-    assert "key: value" not in Path.read_text(Path(asset))
+    assert "one: value" not in Path.read_text(Path(asset))
```

Comment:
`Path.read_text(Path(asset))` -> `Path(asset).read_text()`

Reply:
Nice, I'll change it at all places, thanks.